build.sbt

```diff
@@ -11,36 +13,38 @@ lazy val core = project
   .settings(commonSettings)
   .settings(
     name := "sbt-http4s-org",
-    addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.4.6"),
-    addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.9.4"),
+    addSbtPlugin("com.codecommit" % "sbt-spiewak" % "0.17.0"),
     addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.15"),
     addSbtPlugin("com.lightbend.sbt" % "sbt-java-formatter" % "0.6.0"),
-    addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.8.1"),
-    addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0"),
-    addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.6.0"),
     addSbtPlugin("io.chrisdavenport" % "sbt-mima-version-check" % "0.1.2"),
-    addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
+    addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2"),
   )
 
 // General Settings
 lazy val commonSettings = Seq(
-  scalaVersion := "2.12.12",
-  crossScalaVersions := Seq(scalaVersion.value),
-  libraryDependencies ++= Seq(
-  )
+  crossScalaVersions := Seq("2.12.12"),
+  scalaVersion := crossScalaVersions.value.head,
+  headerLicenseStyle := LicenseStyle.SpdxSyntax,
+  headerLicense := {
+    val current = java.time.Year.now().getValue
+    val copyrightYear = startYear.value.fold(current.toString)(start => s"$start-$current")
+    LicenseDetection(
+      licenses.value.toList,
+      organizationName.value,
+      Some(copyrightYear),
+      headerLicenseStyle.value
+    )
+  }
```

Comment:
I like these abbreviated headers enough to have left them in Http4sOrgPlugin.  I guess I'd concede to the full mess in the spirit of unity.

Reply:
The defaults aren't _that_ bad.